controllers/statusreport/statusreport_adapter.go

```diff
@@ -197,7 +204,7 @@ func (a *Adapter) EnsureSnapshotFinishedAllTests() (controller.OperationResult,
 				a.logger.Error(err, "Failed to Update Snapshot AppStudioTestSucceeded status")
 				return controller.RequeueWithError(err)
 			}
-			a.logger.LogAuditEvent("Snapshot integration status condition marked as passed, all Integration PipelineRuns succeeded",
+			a.logger.LogAuditEvent(fmt.Sprintf("Snapshot integration status condition marked as passed, all of %d Integration PipelineRuns succeeded", len(*integrationTestScenarios)),
```

Comment:
I realized that integration tests in this context are only **required** integration tests, optional tests are not evaluated, so maybe wording everywhere should be updated. But mainly in this place.
```suggestion
			a.logger.LogAuditEvent(fmt.Sprintf("Snapshot integration status condition marked as passed, all of %d required Integration PipelineRuns succeeded", len(*integrationTestScenarios)),
```